### PR TITLE
Jupyterlab tutorial

### PR DESCRIPTION
**Pull request by annefou**
In reference with issue #1730 
I have started a short tutorial. It is not finished but this PR is mostly to indicate that I am working on it and be happy to get contributions/help/feedback.

### CODE REVIEW

**Comment by annefou** [comment]
It is a very short tutorial; very generic on the usage of JupyterLab in Galaxy.

**Comment by bgruening** [comment]
@annefou ready for review?

**Comment by annefou** [comment]
Yes. I would be happy to get feedback.

File: topics/galaxy-ui/tutorials/jupyterlab/tutorial.md [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+- "Import libraries"
+- "Use get() to import datasets from your history to the notebook"
+- "Use put() to export datasets from the notebook to your history"
+- "Save your notebook into your history"
+follow_up_training:
+-
+    type: "internal"
+    topic_name: introduction
+    tutorials:
+        - r-basics
```
Comment (bgruening) [inline topics/galaxy-ui/tutorials/jupyterlab/tutorial.md]: is this still valid?

Comment (annefou) [reply topics/galaxy-ui/tutorials/jupyterlab/tutorial.md]: Yes. It makes little sense to have a follow up tutorial for R basics only. I guess in the future, we could have follow up tutorials for each supported kernels but for now I remove it.

**Comment by annefou** [comment]
> Thanks @annefou
> 
> ecosystem_min.png ...is not needed in this PR isn't it?

Definitely not! I am not sure where it comes from. I guess a messy merge at some point.

**Comment by annefou** [comment]
Thanks for sorting out my mess!

**Comment by bgruening** [comment]
Good luck tomorrow!

**Pull request merged**